packages/frontend/src/components/Info/TimeLeft.tsx

```diff
@@ -13,8 +13,8 @@ export const TimeLeft = () => {
   const [timeLeft, setTimeLeft] = useState(formatTimeLeft(timestamp))
 
   useEffect(() => {
-    const interval = setTimeoutImmediately(() => setTimeLeft(formatTimeLeft(timestamp)), 1_000)
-    return () => clearTimeout(interval)
+    const interval = setIntervalImmediately(() => setTimeLeft(formatTimeLeft(timestamp)), 1_000)
+    return () => clearInterval(interval)
   }, [timestamp, timeLeft])
```

Comment:
If we're setting an interval instead of timeout, I think it would be best to remove `timeLeft` from the dependency array, otherwise it'll spawn a new interval every time `timeLeft` updates. Basically interval works cyclically so we just want to set it once when the timestamp is known.

Reply:
Yep, removed it.